Pod/Classes/KolodaView/KolodaView.swift

```diff
@@ -11,10 +11,10 @@ import pop
 
 //Default values
 private let defaultCountOfVisibleCards = 3
-private let backgroundCardsTopMargin: CGFloat = 4.0
-private let backgroundCardsScalePercent: CGFloat = 0.95
-private let backgroundCardsLeftMargin: CGFloat = 8.0
-private let backgroundCardFrameAnimationDuration: NSTimeInterval = 0.2
+private let defaultBackgroundCardsTopMargin: CGFloat = 4.0
```

Comment:
This constants are used in `frameForCardAtIndex` as default implementation. So it wasn't meant to be adjustable, because user of Koloda can apply totally different pattern for frames calculation and we'll end up with 4 extra variables.